_includes/masthead.html

```diff
@@ -11,7 +11,9 @@
         {% unless logo_path == empty %}
           <a class="site-logo" href="{{ '/' | relative_url }}"><img src="{{ logo_path | relative_url }}" alt=""></a>
         {% endunless %}
-        <a class="site-title" href="{{ '/' | relative_url }}">{{ site.masthead_title | default: site.title }}</a>
+        <a class="site-title" href="{{ '/' | relative_url }}">{{ site.masthead_title | default: site.title }}
+        {% if site.subtitle %}<div class="site-subtitle">{{ site.description }}</div>{% endif %}
```

Comment:
Any particularly reason you're using `site.subtitle` as the conditional, but displaying `site.description`? Wouldn't it make more sense to output `site.subtitle`?

A site description and site subtitle might be different. I would favor keeping things consistent.
If your'e checking for `site.subtitle` output that if `true`. Otherwise use `{% if site.description%}<div class="site-description">{{ site.description }}</div>{% endif %}` makes more sense to me.

Though that might not be intended by people who have a description filled out for SEO `<meta>` tag reasons but don't want it shown in masthead.